packages/opentelemetry-browser-extension-autoinjection/package.json

```diff
@@ -26,7 +26,7 @@
     "@opentelemetry/api": "^1.0.2"
   },
   "devDependencies": {
-    "@opentelemetry/api": "1.0.2",
+    "@opentelemetry/api": "1.1.0",
```

Comment:
below quite some deps are pinned to fixed versions (github doesn't allow to comment there). I think we should ranges there.

Reply:
Changed those as well.